README.md

```diff
@@ -1,3 +1,6 @@
+[![Last image-template](https://img.shields.io/badge/last%20template%20update-v0.1.0-informational)](https://github.com/Tecnativa/image-template/tree/v0.1.0)
```

Comment:
I'm sorry to interrupt this so early, but the README modifiation here is from my PoV unacceptable. If you'd like to link to the package, please do it somewhere further down below, and especially _below_ the heading. Please do not use bades for this as well, as the badges at the top shall remain the only ones. The miscellaneous section is in my eyes appropriate,